src/flows/shared/steps/OrganizeCollection/CollectionCreateOrEditForm.tsx

```diff
@@ -17,14 +17,15 @@ import { Collection, CollectionLayout } from 'types/Collection';
 import useAuthenticatedGallery from 'hooks/api/galleries/useAuthenticatedGallery';
 import useCreateCollection from 'hooks/api/collections/useCreateCollection';
 import Mixpanel from 'utils/mixpanel';
-import { Nft } from 'types/Nft';
+import { StagingItem } from './types';
```

Comment:
same here, most of the changes in this file are to change the EditModeNft type to StagingItem